docs/notebooks/data_transformation.pct.py

```diff
@@ -244,6 +255,71 @@ def normalise(x, mean=None, std=None):
 print(f"observation: {observations[arg_min_idx, :]}")
 
 
+# %% [markdown]
+# # Data transformation with the DataTransformModelWrapper class
+#
+# Actually, we can achieve the same effect with a standard `BayesianOptimizer` by using a `DataTransformModelWrapper`.
+#
+# First, we create a new model class that inherits from the model wrapper class and the kind of `TrainableProbabilisticModel` that we want to use.
+
+# %%
+from trieste.models.normalization import DataTransformModelWrapper
+
+class GPRwithDataNormalization(DataTransformModelWrapper, GaussianProcessRegression):
+    pass
+
+
+# %% [markdown]
+#
+# Now we create `DataTransformer`s for query_points and observations, and pass these into our new model class when we construct it.
+
+# %%
+from trieste.models.normalization import StandardTransformer
+
+search_space = TRID_10_SEARCH_SPACE  # Restore the original search space
+
+# Create transformers and pass these in.
+query_point_transformer = StandardTransformer(initial_data.query_points)
+observation_transformer = StandardTransformer(initial_data.observations)
+# Note that the underlying model is expected to have normalized data when passed in.
+normalized_data = Dataset(
+    query_point_transformer.transform(initial_data.query_points),
+    observation_transformer.transform(initial_data.observations)
+)
+model = GPRwithDataNormalization(
+    model=build_gp_model(normalized_data),
+    optimizer=Optimizer(
+        gpflow.optimizers.Scipy(),
+        minimize_args={"options": dict(maxiter=100)}
+    ),
+    num_kernel_samples=100,
+    query_point_transformer=query_point_transformer,
+    observation_transformer=observation_transformer,
+)
+
+# %% [markdown]
+#
+# Then we run Bayesian Optimization as usual and display the results.
+#
+# Again, normalization has improved BO performance.
+
+
+# %%
+bo = trieste.bayesian_optimizer.BayesianOptimizer(observer, search_space)
+result = bo.optimize(num_steps, initial_data, model)
+dataset = result.try_get_final_dataset()
+
+query_points = dataset.query_points.numpy()
+observations = dataset.observations.numpy()
+
+arg_min_idx = tf.squeeze(tf.argmin(observations, axis=0))
+
+plot_regret_with_min(dataset)
+
+print(f"query point: {query_points[arg_min_idx, :]}")
+print(f"observation: {observations[arg_min_idx, :]}")
+
+
```

Comment:
lets add a small paragraph explaining you can be more sophisticated - just in few words how, and then point to the more advanced notebook